### PR TITLE
Migrated node commands to python

### PR DESCRIPTION
**Pull request by vivek-kumar-pandey**
#2 

### Types of changes

- [ ] Bug fix (a change which fixes an issue)
- [x] New feature (a change which adds functionality)

### Short description of what this resolves:

As we decided to go with python for consistency, converting node files to python

### Checklist:

- [x] I have performed a self-review of my own code.
- [ ] The code follows the style guidelines of this project.
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+# Add env variables here
```
Comment (GauravGusain98) [inline .env.example:1]: How is this file working?

Comment (vivek-kumar-pandey) [reply .env.example:1]: Currently we don't have anything in it. But environment variables will be added here as required.

Comment (GauravGusain98) [reply .env.example:1]: I couldn't see any place from where we are fetching environment variables

Comment (vivek-kumar-pandey) [reply .env.example:1]: Yes not currently, but will be required later

**Review by Satendra-SR — APPROVED** [review]
Looks good in terms of scaffolding.

**Pull request merged**